staging/src/k8s.io/kube-proxy/config/v1alpha1/types.go

```diff
@@ -250,10 +250,10 @@ type KubeProxyConfiguration struct {
 
 // ProxyMode represents modes used by the Kubernetes proxy server.
 //
-// Currently, two modes of proxy are available on Linux platforms: 'iptables' and 'ipvs'.
-// One mode of proxy is available on Windows platforms: 'kernelspace'.
+// Currently, three modes of proxy are available on Linux platforms: 'iptables', 'ipvs',
+// and 'nftables'. One mode of proxy is available on Windows platforms: 'kernelspace'.
```

Comment:
Even better (but perhaps for a different PR):
```suggestion
// Three modes of proxy are available on Linux platforms: `iptables`, `ipvs`,
// and `nftables`. One mode of proxy is available on Windows platforms: `kernelspace`.
```

This will render better into https://kubernetes.io/docs/reference/config-api/kube-proxy-config.v1alpha1/#kubeproxy-config-k8s-io-v1alpha1-ProxyMode